maintainers/maintainer-list.nix

```diff
@@ -110,6 +110,12 @@
     githubId = 1174810;
     name = "Nikolay Amiantov";
   };
+  aberDerBart = {
```

Comment:
Please include in separate commit.

Reply:
is there a convention for commit messages on adding maintainers? I couldn't find any, had a look at `maintainer-list.nix` and saw that some people added themselves as maintainers along with adding a package so I thought this was the way...

Reply:
I'm not sure if this is documented in the nixpkgs manual or not but a separate commit is the preferred way. Sometimes reviewers miss this or just don't know. Always hard to enforce uniformity on a large project like this... but we try.

Reply:
okay, i went with `maintainers/aberDerBart: init`, it seemed consistent to the other message templates.